src/Diagnostics.ModelsAndUtils/Models/QueryResponse.cs

```diff
@@ -1,8 +1,11 @@
-﻿namespace Diagnostics.ModelsAndUtils.Models
+﻿using System.Collections.Generic;
+
+namespace Diagnostics.ModelsAndUtils.Models
 {
     public class QueryResponse<T>
     {
         public CompilerResponse CompilationOutput;
+        public IEnumerable<RuntimeLogEntry> RuntimeOutput;
```

Comment:
nit: probably rename this to be RuntimeLogOutput

Reply:
Renamed.